src/components/label/Label.test.tsx

```diff
@@ -13,4 +13,11 @@ describe(`Label component`, () => {
     expect(container).toMatchSnapshot()
     expect(await axe(container)).toHaveNoViolations()
   })
+  it('renders an asterisk if required', async () => {
```

Comment:
```suggestion

  it('renders an asterisk if required', async () => {
```

It is a small nitpick, sorry